packages/editor/src/components/local-autosave-monitor/index.js

```diff
@@ -62,6 +57,7 @@ function useAutosaveNotice() {
 		} ),
 		[]
 	);
+	const { getEditedPostAttribute } = useSelect( 'core/editor' );
```

Comment:
You forgot to remove `getEditedPostAttribute` from the previous `useSelect` call. :)

Reply:
Oops